model/tactile_markers.urdf.xacro

```diff
@@ -42,6 +42,10 @@
 			<parent link="${prefix}${finger}_middle_link"/>
 			<tactile channel="${channel}">
 				<xacro:taxel mesh="${F}mo" idx="${tactMap.get(F+'mo', -1)}"/>
+				<!--tmm does not exist for L1 thumb, but can be tmo which is larger and contains tmm -->
+				<xacro:if value="${layout=='L1' and finger=='thumb'}">
+					<xacro:taxel mesh="${F}mm" idx="${tactMap.get(F+'mo', -1)}"/>
+				</xacro:if>
 				<xacro:taxel mesh="${F}mm" idx="${tactMap.get(F+'mm', -1)}"/>
```

Comment:
```suggestion
				<!--tmm does not exist for L1, but is covered by tmo as well -->
				<xacro:taxel mesh="${F}mm" idx="${tactMap.get(F+'mm', tactMap.get(F+'mo', -1) if layout=='L1' else -1)}"/>
```

Reply:
none of the text are clear in fact. and I don't find you to do another suggestion or reject this one...

Reply:
it took me some time to ensure this optimization of the code is doing the same due mentally needing to evaluate the code to understand what it does, while the if case is straight forward because it explicitly refers to the thumb. 

With the optimization we have to access an external document, to verify that there is no other ${F}mm that is not missing in another finger to be sure it works only for the thumb. For me, this opacifies  the understanding for future developments, and would for sure be missed in case one new layout has a pinky tmm missing or so. 

I confirm it should do the job for now, but don't like it.